tests/conftest.py

```diff
@@ -0,0 +1,43 @@
+"""Common pytest fixtures."""
+
+from __future__ import (division, absolute_import, print_function,
```

Comment:
There are similar boilerplate includes in a few of the python modules. This isn't wrong but it an be misleading. For instance I was wondering where future print and division were being used in this module.

Do we, as a team, have an opinion on this?